Composer/packages/client/src/pages/design/exportSkillModal/content/AddCallers.tsx

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { SharedColors } from '@uifabric/fluent-theme';
+import formatMessage from 'format-message';
+import { ActionButton, css, FontWeights, TextField } from 'office-ui-fabric-react';
+import React from 'react';
+import { tableColumnHeader, tableRow, tableRowItem } from '../../../botProject/styles';
+import { ContentProps } from '../constants';
+
+const header = css`
+  display: flex;
+  flex-direction: row;
+  height: 42px;
+`;
+
+const addNewAllowCallers = {
+  root: {
+    fontSize: 12,
+    fontWeight: FontWeights.regular,
+    color: SharedColors.cyanBlue10,
+    paddingLeft: 0,
+    marginLeft: 5,
+  },
+};
+
+const removeCaller = {
+  root: {
+    fontSize: 12,
+    fontWeight: FontWeights.regular,
+    color: SharedColors.cyanBlue10,
+    paddingLeft: 0,
+    paddingBottom: 5,
+  },
+};
+
+export const AddCallers: React.FC<ContentProps> = ({ projectId, callers, setCallers }) => {
+  const handleRemove = (index) => {
+    var currentCallers = callers.slice();
```

Comment:
maybe \[...callers\].splice(index, 1)?

Also can we avoid using var?

Reply:
Fixed